### ISSUE TITLE
Community Diligence Review of ND Labs Enterprise Allocator

### ISSUE THREAD

**Issue by filecoin-watchdog** [filecoin-project/Allocator-Governance#109]
Allocator Compliance Review: https://compliance.allocator.tech/report/f03012747/1721402091/report.md

Allocator Application: https://github.com/filecoin-project/notary-governance/issues/1025

**First example**: https://github.com/NDLABS-Leo/Allocator-Pathway-Enterprise-data/issues/3

KYB was asked for 3 months after the first DataCap allocation. https://github.com/NDLABS-Leo/Allocator-Pathway-Enterprise-data/issues/3.  

Client asking 10 copies at 10PiBs. 

Gov Team needs to follow up proof of KYB and dataset size

**Second example**: https://github.com/NDLABS-Leo/Allocator-Pathway-Enterprise-data/issues/8

Client asking 10 copies at 10PiBs. Yet only 3-4SPs?

Gov Team needs to follow up to confirm Dataset size and proof of KYB.

**Comment by NDLABS-Leo** [comment]
`KYB was asked for 3 months after the first DataCap allocation`

Because our Allocator has been buggy, we waited a long time in the middle of the process, and in the community need to carry out spark compatibility as soon as the results came out the first time we asked the customer to carry out the sprak retrieval rate enhancement.

![image](https://github.com/user-attachments/assets/89f96cde-dadf-4fec-957b-818acdef769f)
![image](https://github.com/user-attachments/assets/79e3b153-e4a4-4eae-b382-b9c6774439a5)

**Comment by NDLABS-Leo** [comment]
`Client asking 10 copies at 10PiBs. Yet only 3-4SPs?`

Since the client is only assigned the first round, we require at least 4 nodes for the first round, and the number of nodes will increase as the project progresses.
![image](https://github.com/user-attachments/assets/bb43d7a4-c246-4b62-a27e-8ec2b9f0110c)

**Comment by NDLABS-Leo** [comment]
**`1. Client's kyc/kyb attestation records`**

<The Second Life> - <3D-printed>
![image](https://github.com/user-attachments/assets/67a2c98b-171a-4244-aaac-52352742399f)

<venture capital Cabin> - <enterprise data>
![image](https://github.com/user-attachments/assets/53c83a84-7639-406b-a8f6-eae7e1b93644)

 <AINN Layer2> - <on-chain data>
![image](https://github.com/user-attachments/assets/08235969-2e78-494f-90f5-78ef06992b6a)

**Comment by NDLABS-Leo** [comment]
`Proof of customer's node usage`

AINN Layer2 - on-chain data (A total of 9 nodes are in use)

![image](https://github.com/user-attachments/assets/65902084-38de-4339-bba4-4e8237089e62)

venture capital Cabin - enterprise data (Since this is the first round of usage, there are a total of 4 nodes in use)
![image](https://github.com/user-attachments/assets/95283918-674d-458d-b5f6-c0d90a52512d)

The Second Life - 3D-printed new2(Since this is the first round of usage, there are a total of 5 nodes in use)
![image](https://github.com/user-attachments/assets/48e1b2db-becb-4e04-be72-9f07d76bb3af)

**Comment by NDLABS-Leo** [comment]
`On the node retrieval rate enhancement`

![image](https://github.com/user-attachments/assets/401e9483-1098-47ca-a955-3b17c136a3cc)
![image](https://github.com/user-attachments/assets/9fb87964-0902-4739-84e9-56778b13afa4)
![image](https://github.com/user-attachments/assets/6f262c69-9b73-47b7-b795-a6857023adb6)
![image](https://github.com/user-attachments/assets/e6c47b28-4a01-4ec2-baad-d35883802955)
![image](https://github.com/user-attachments/assets/a0fd3817-0d88-48a0-a4b4-8c316b403a83)

**Comment by galen-mcandrew** [comment]
**Flagging and pausing this compliance review, given some confusing details that need to be addressed.**

This allocator has two pathways, which is reasonable and encouraged under the current program structure. However, there appears to be a lot of overlap between these pathways, without clear specialization. Specifically, [this client](https://github.com/NDLABS-Leo/Allocator-Pathway-Enterprise-data/issues/3) has an almost identical application across both pathways for the same set of data.

Additionally, this client has encrypted unretrievable data, with no evidence of paid enterprise deal-making to show program alignment. Their SP distribution is also noncompliant with the [allocator's application](https://github.com/filecoin-project/notary-governance/issues/1025), regarding number, region, and VPN usage. 

Other clients in this pathway are also not engaging in compliant SP deal making and distribution, such as here: https://check.allocator.tech/report/NDLABS-Leo/Allocator-Pathway-Enterprise-data/issues/8/1723003166749.md

@NDLABS-Leo Given the issues uncovered in this investigation, we need to see a clear and concise explanation of the difference between these two allocator pathways. 

Please provide any additional evidence and diligence for our review regarding the above issues.

**Comment by NDLABS-Leo** [comment]
@galen-mcandrew @Kevin-FF-USA 

Thank you for flagging this and giving us time to explain.

1. **Regarding collaboration with individual clients**:  
We have closed applications that do not fit our channel, which helps us manage our applications better. Additionally, because AINN has a large data volume, we had in-depth discussions about their project back in March. Our supplemental quota was only 2.5P, and after it was exhausted, we did not promote it externally. If we supplement the quota again, we will find ways to promote it, allowing more compliant clients to participate.
![image](https://github.com/user-attachments/assets/a524a301-d600-4b3b-a539-1059c8238f85)
![image](https://github.com/user-attachments/assets/fe940c80-d592-4b23-8045-8de30f23059c)


2. **Proof of data quality**:  
During the first round of the client’s quota distribution, we required them to undergo data storage testing. We downloaded and sampled the data, and after verifying that the data content matched the client’s description, we proceeded with the distribution.
![image](https://github.com/user-attachments/assets/f66cacc8-dba0-4471-8b29-2a57be7f6137)
![image](https://github.com/user-attachments/assets/209a612f-dda7-4ff4-a91f-56e01078e8f6)

3. **Regarding the client's SP location requirements**:  
The client is currently using 10 nodes, primarily distributed across HK/KR/CN Guangdong Shenzhen/CN Sichuan Chengdu. We believe this meets the requirements for different regions. We also reviewed the operators of the nodes; there are five types of operators. Additionally, the client plans to add more nodes in future projects.
![image](https://github.com/user-attachments/assets/010cb334-ad73-489a-9296-ad04e3ab2814)


4. **Regarding differences between distributor pathways**:  
NDCloud is a public cloud aggregation platform, and the clients we aggregate to the public cloud have real and substantial data storage needs. When clients use NDCloud’s public cloud aggregation platform, those with storage needs can apply through NDCloud. Typically, they are web2 and web3 enterprise data owners.

Enterprise data - HK Cyberport pathway is primarily an application channel for incubating, accelerating, and investing in web3.0 startups. Cyberport is an innovative digital community with nearly 900 digital technology companies, managed by the Hong Kong Cyberport Management Company Limited, which is wholly owned by the Government of the Hong Kong Special Administrative Region. This channel is mainly for clients that have passed Cyberport’s selection process.

**Comment by NDLABS-Leo** [comment]
https://github.com/filecoin-project/Allocator-Governance/issues/109#issuecomment-2259794902

https://github.com/filecoin-project/Allocator-Governance/issues/109#issuecomment-2259804177

https://github.com/filecoin-project/Allocator-Governance/issues/109#issuecomment-2272551951

https://github.com/filecoin-project/Allocator-Governance/issues/109#issuecomment-2272564304

https://github.com/filecoin-project/Allocator-Governance/issues/109#issuecomment-2272660045

**Comment by NDLABS-Leo** [comment]
@galen-mcandrew @Kevin-FF-USA 
During the period when this review request was flagged and paused, we held a Zoom meeting with Galen. He described in detail some of his questions and concerns about our channel at the meeting, and we responded to his concerns. 
To clarify the differences between the two channels and provide examples of actual clients under each, I will outline the specifics below.

In our allocator application form, apart from how we verify client information, the logic, and standards for reviewing data, sps, and sealing situations are essentially similar across both channels. For more details on our past review process and reasoning, please refer to this link: https://github.com/filecoin-project/Allocator-Governance/issues/131#issuecomment-2350886904. 
Additionally, previous comments on this ticket include details on allocation review  for the clients in this channel.

Now, I’ll focus on explaining why the AINN client applied through both ND channels and the characteristics of all clients in this particular channel.

 1. Client types accepted in this channel:
We work with corporate clients that have partnerships with Hong Kong’s Cyberport and have been vetted through their screening process. Clients are categorized based on the value of their data and the depth of collaboration, which informs their corresponding payment standards.
For more details, please see: https://github.com/filecoin-project/notary-governance/issues/1025#issuecomment-1890592797.
 2. AINN Client info:
AINN has been using NDCloud’s cloud aggerate platform services for data backup and storage since the early stages of their project. They also actively expand in the Hong Kong market and have partnerships with well-known institutions like Cyberport. You can find more details on their Twitter and through their press releases.https://www.chaincatcher.com/article/2119596 
https://x.com/AILayerXYZ
The screenshot in this response shows that we began collaborating in March: https://github.com/filecoin-project/Allocator-Governance/issues/109#issuecomment-2337743741.
Therefore, AINN meets the client requirements for both ND channels. Given their strong desire for collaboration and other specifics, we did not charge them any fees.

Additionally, the other two clients in this channel are:

 1. Cabin VC: Since 2023, Cabin VC has focused on developing virtual asset management businesses in Hong Kong. They have deep collaborative ties with institutions like Cyberport. We have engaged with them both online and offline. Due to the nature of their business, they haven’t widely publicized their activities. You can find an example here: https://mp.weixin.qq.com/s/H8LzdQvBbvn4LCB-VVBX0g. Cabin VC meets the application standards for our channel.
 2. The Second Life: This company specializes in digital hipster blocks, using 3D printing and digital twins as underlying technologies to create metaverse experiences. Their services focus on future-forward private ordering, self-realization, and the seamless creation of a second life for real individuals. After online communication, we confirmed that they are in the application stage with Cyberport’s incubation program. Their significant need for image file storage aligns with ND’s service scope, and we have provided them with data processing service. As per the fee structure mentioned in our application, following their KYB certification, we charged them the equivalent of 500 FIL in USD for the data processing service.
See the illustration attached. 【【在这个地方放付款截图】】

To summarize, this outlines the client situation for this channel. To prevent any future misunderstandings regarding client statuses, we will archive KYB information and records of whether clients meet channel requirements here: https://github.com/NDLABS-Leo/Allocator-Pathway-Enterprise-data/blob/main/README.md.

Thank you for your time and consideration. ND is making every effort to bring more genuine clients into the Filecoin network. Our future work plan is outlined in this commenthttps://github.com/filecoin-project/Allocator-Governance/issues/131#issuecomment-2350886904), and we look forward to the community’s collaboration and guidance.

**Comment by galen-mcandrew** [comment]
Thank you for the additional details. It seems like one of the major components for this pathway is participation in the HK Cyberport program. 

Are there some additional documents or details that could be provided to show compliance and participation in this program? For example, is there a standard and official indication for these "900 digital technology companies, managed by the Hong Kong Cyberport Management Company Limited"?

Additionally, given the nature of this enterprise pathway, we would love to see further evidence of paid storage deals. 

In the meantime, we are requesting an additional 5PiB for this allocator pathway, with the expectation that we can see increased evidence of specialization of this pathway. For example, I strongly advise your team to partition the clients and datasets that are working with this pathway compared to other ND Labs and other allocators in general.

**Comment by Kevin-FF-USA** [comment]
DataCap refreshed.
https://datacapstats.io/notaries/f03012747
